src/F1_plotting.py

```diff
@@ -171,95 +162,220 @@ def parse_simulation_log(path_log_file=None, log_type="ERROR"):
     return logs_dict
 
 
-def draw_graph(
-    dict_values,
-    energysystem,
-    user_input,
-    edge_labels=True,
-    node_color="#eeac7e",
-    edge_color="#eeac7e",
-    show_plot=True,
-    save_plot=True,
-    node_size=5500,
-    with_labels=True,
-    arrows=True,
-    layout="dot",
-):
-    """
+def fixed_width_text(text, char_num=10):
+    """Add linebreaks every char_num characters in a given text.
 
     Parameters
     ----------
-    dict_values: dict
-        Dict with all simulation parameters
-    energysystem:
-    user_input: dict
-    edge_labels: bool
-        Default: True
-    node_color: str
-        Default: "#eeac7e"
-    edge_color: str
-        Default: "#eeac7e"
-    show_plot: bool
-        Default: True
-    save_plot: bool
-        Default: True
-    node_size: int
-        Default: 5500
-    with_labels: bool
-        Default: True
-    arrows: bool
-        Default: True
-    layout: str
-        Default: "dot"
-
+    text: obj:'str'
+        text to apply the linebreaks
+    char_num: obj:'int'
+        max number of characters in a line before a line break
+        Default: 10
     Returns
     -------
+    obj:'str'
+        the text with line breaks after every char_num characters
 
     """
+    # total number of characters in the text
+    text_length = len(text)
+    # integer number of lines of `char_num` character
+    n_lines = int(text_length / char_num)
+    # number of character in the last line
+    last_line_length = text_length % char_num
+
+    # split the text in lines of `char_num` character
+    split_text = []
+    for i in range(n_lines):
+        split_text.append(text[(i * char_num) : ((i + 1) * char_num)])
+
+    # I if the last line is not empty
+    if n_lines > 0:
+        if last_line_length > 0:
+            split_text.append(text[((i + 1) * char_num) :])
+        answer = "\n".join(split_text)
+    else:
+        answer = text
+    return answer
+
+
+class ESGraph:
+    def __init__(
+        self,
+        energy_system=None,
+        filepath="network",
+        img_format=None,
+        legend=True,
+        txt_width=10,
+        **kwargs
+    ):
+        """Draw the energy system with Graphviz.
+
+        Parameters
+        ----------
+        energy_system: `oemof.solph.network.EnergySystem`
+            The oemof energy stystem
+
+        filepath: str
+            path, where the rendered result shall be saved, if an extension is provided, the format
+            will be automatically adapted except if the `img_format` argument is provided
+            Default: "network"
+
+        img_format: str
+            extension of the available image formats of graphviz (e.g "png", "svg", "pdf", ... )
+            Default: "pdf"
+
+        legend: bool
+            specify, whether a legend will be added to the graph or not
+            Default: False
+
+        Returns
+        -------
+        None: render the generated dot graph in the filepath
```

Comment:
```suggestion
        None: render the generated dot graph in the filepath
        
        Notes
        --------
        When new oemof-solph asset types are added to the available types in the MVS, this function needs to be updated, so that it can render the asset in the graph.
```